Calculator/FormulaTests/FormulaTests.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  FormulaTests.swift
+//  FormulaTests
+//
+//  Created by Hemg on 2023/06/07.
+//
+
+import XCTest
+@testable import Calculator
+
+final class FormulaTests: XCTestCase {
+    
+    var sut: Formula!
+    var operands: CalculatorItemQueue<Double>!
+    var operators: CalculatorItemQueue<Operator>!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        operands = CalculatorItemQueue<Double>()
+        operators = CalculatorItemQueue<Operator>()
```

Comment:
```suggestion
        let operands = CalculatorItemQueue<Double>()
        let operators = CalculatorItemQueue<Operator>()
```

전역 변수인 operands, operators을 없애고 위와 같이 변경하면 좋을 것 같아요.